packages/datetime/src/dateInput.tsx

```diff
@@ -100,10 +101,16 @@ export interface IDateInputProps extends IDatePickerBaseProps, IProps {
     /**
      * The position the date popover should appear in relative to the input box.
      * @default Position.BOTTOM
+     * @deprecated since v1.15.0, use `popoverProps.position`
```

Comment:
Note: We should add this message to `DateRangeInput` in https://github.com/palantir/blueprint/pull/940 and also `NumericInput` if we move that component to support `IInputProps`. (Separate PRs of course)